torch/testing/_internal/common_quantization.py

```diff
@@ -356,16 +357,18 @@ def is_leaf_module(module):
                     submodule_name_count += 1
             return submodule_name_count == 0
 
+        quant_type = getattr(getattr(module, 'qconfig', None), 'quant_type', None)
```

Comment:
please use https://github.com/pytorch/pytorch/blob/master/torch/quantization/fx/utils.py#L221 instead (will probably move this function to a common utils.py file later)